crates/evm/src/executor/inspector/tracer.rs

```diff
@@ -116,30 +114,26 @@ impl Tracer {
             self.step_stack.pop().expect("can't fill step without starting a step first");
         let step = &mut self.traces.arena[trace_idx].trace.steps[step_idx];
 
-        if let Some(pc) = interp.program_counter().checked_sub(1) {
```

Comment:
This is always `Some` since it's called in `step_end`, meaning that at least 1 opcode was evaluated.